deploy/examples/operator-openshift.yaml

```diff
@@ -449,7 +449,6 @@ data:
   #        memory: 256Mi
   #        cpu: 100m
 
-
```

Comment:
I have no idea why this line was deleted. I _think_ this is one of the projects I'm touching right now that has a linter that causes VS Code to rewrite files when I save. I need to ask someone else how to stop that...

Reply:
I don't see an issue deleting extra empty lines...and vs code on r`ctrl+s` auto format which could be the reason but if you don't want to auto-format while saving you can try ctrl+k+s